src/elexmodel/handlers/data/VersionedData.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
Should this be `from elexmodel.logger import getModelLogger`? 🤔

Reply:
yeah it should, good catch, thanks. unclear how that happened